packages/wallets/src/features/cfd/modals/CTraderAddAccountSuccessModal/CTraderAddAccountSuccessModal.scss

```diff
@@ -0,0 +1,24 @@
+.wallets-ctrader-account-add-success-modal {
+    max-width: 44rem;
+    padding: 2.4rem;
+    display: flex;
+    flex-direction: column;
+    gap: 2.4rem;
+    background-color: var(--general-main-1, #ffffff);
+    border-radius: 0.8rem;
+
+    @include mobile {
+        width: 100%;
+        padding-inline: 1.6rem;
+        gap: 1.6rem;
+    }
+}
+
+.wallets-ctrader-success-icon {
+    position: relative;
+    &-check {
+        position: absolute;
+        bottom: 0;
+        right: -24px;
+    }
+}
```

Comment:
the method is correct however the writing style is abit different. It should be:
```
.wallets-ctraders-success-icon {
    /* CSS Code*/
   &--check {
      /* CSS Code*/
   }
}
```